### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ihorsokolenko**
https://qamar24omalinevska.atlassian.net/browse/TP-258?atlOrigin=eyJpIjoiYjA0ZTI5MGU5OTM4NDJkNDkxNTUyYTcwMDRkZDg2YjUiLCJwIjoiaiJ9

### CODE REVIEW

**Review by malinevska — COMMENTED** [review]
GJ

File: cypress/e2e/hermioneAccount.cy.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  it('should provide the ability to work with Hermione_s bank account', () => {
+    cy.contains('.btn', customerLoginBtn).click();
+    cy.url().should('contain', 'customer');
+    cy.get('#userSelect').select(user);
+    cy.get('[type="submit"]').click();
+
+    cy.get('.fontBig').should('contain', user);
+    cy.get('#accountSelect').should('contain.text', '1001');
+    cy.contains('[ng-hide="noAccount"]', 'Balance')
+      .contains('strong', '0')
```
Comment (djtyrf312) [inline cypress/e2e/hermioneAccount.cy.js:21]: what does it check?
```suggestion
      .contains('strong', '0')
```

File: cypress/e2e/hermioneAccount.cy.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+      .should('be.visible');
+
+
+    cy.contains('.ng-binding', 'Dollar').should('be.visible');
+    cy.get('[ng-click="deposit()"]').click();
+    cy.findByPlaceholder('amount').should('be.visible').type(depositAmount);
+    cy.contains('[type="submit"]', 'Deposit').click();
+    cy.get('[ng-show="message"]').should('contain', 'Deposit Successful');
+    cy.contains('[ng-hide="noAccount"]', 'Balance')
+      .contains('strong', depositAmount)
```
Comment (djtyrf312) [inline cypress/e2e/hermioneAccount.cy.js:31]: check on ui mode for this code. it doesn't check the expected balance.
```suggestion
      .contains('strong', depositAmount)
```

File: cypress/e2e/hermioneAccount.cy.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    cy.get('[ng-click="withdrawl()"]').click();
+    cy.contains('[type="submit"]', 'Withdraw').should('be.visible');
+    cy.get('[placeholder="amount"]').type(withdrawAmount);
+    cy.contains('[type="submit"]', 'Withdraw').click();
+    cy.get('[ng-show ="message"]')
+      .should('contain', 'Transaction successful')
+      .should('be.visible');
+
+    cy.contains('[ng-hide="noAccount"]', 'Balance')
+      .contains('strong', '0')
```
Comment (djtyrf312) [inline cypress/e2e/hermioneAccount.cy.js:43]:
```suggestion
      .contains('strong', '0')
```
this doesn't check the balance after withdrawal

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
check the comments in the pr

**Pull request closed without merging**